### PR TITLE
`ImplicitFreeSurface` default in `HydrostaticFreeSurfaceModel`

### PR DESCRIPTION
**Pull request by simone-silvestri**

### CODE REVIEW

**Review by glwagner — APPROVED** [review]

**Comment by glwagner** [comment]
Should we set gravitational acceleration explicitly in the default?

File: src/ImmersedBoundaries/grid_fitted_immersed_boundaries.jl [hunk 1]
```diff
@@ -44,8 +44,8 @@ end
 GridFittedBottom(bottom_height) = GridFittedBottom(bottom_height, CenterImmersedCondition())
 
 function Base.summary(ib::GridFittedBottom)
-    hmax = maximum(ib.bottom_height)
-    hmin = minimum(ib.bottom_height)
+    hmax = maximum(parent(ib.bottom_height))
+    hmin = minimum(parent(ib.bottom_height))
```
Comment (glwagner) [inline src/ImmersedBoundaries/grid_fitted_immersed_boundaries.jl:48]: What's the issue?

Comment (simone-silvestri) [reply src/ImmersedBoundaries/grid_fitted_immersed_boundaries.jl:48]: scalar indexing when `ib.bottom_height` is an `OffsetArray{CuArray}`

Comment (glwagner) [reply src/ImmersedBoundaries/grid_fitted_immersed_boundaries.jl:48]: Ah, I forgot that it wasn't a field.

**Pull request merged**